src/components/summit-event-bulk-actions/summit-event-bulk-editor-item.js

```diff
@@ -124,7 +140,18 @@ class SummitEventBulkEditorItem extends React.Component
         this.props.onActivityCategoryLocalChanged(this.props.index, activityCategory, isValid)
     }
     onDurationLocalChanged(ev) {
-        let duration = parseInt(ev.target.value);        
+        let duration = Number.isInteger(parseInt(ev.target.value)) ? parseInt(ev.target.value) : null
+        if(duration !== null) {
+            let { event, currentSummit } = this.props;
+            let eventModel = new SummitEvent(event, currentSummit);
+            if(event.start_date) {
+                const end_date = event.start_date + duration;
+                this.props.onEndDateChanged(this.props.index, end_date, eventModel.isValidEndDate(end_date));
+            } else if (event.end_date) {
```

Comment:
this part is not needed duration only sets the end date if it has set and start date